cmd/inspect.go

```diff
@@ -194,7 +272,6 @@ func generateTableWithKeys(writer io.Writer, keys ...string) *tablewriter.Table
 	table.SetHeader(hdrs)
 	table.SetAlignment(tablewriter.ALIGN_CENTER)
 	table.SetColumnAlignment(aligns)
-	table.SetAutoMergeCells(true)
 	table.SetRowLine(false)
```

Comment:
I believe this is used in more places than I see added `SetAutoMergeCells(false)` additions in this PR. Could you check for other refs, please?

Perhaps we could even leave this unchanged and overwrite `t.SetAutoMergeCells(false)` where that is needed instead; but after the call to `generateTableWithKeys()`?

Reply:
On a second search, I find only the two usages. Perhaps you're thinking about `internal/presentation/presentation.go#generateTableWithKeys`.
But I can explicitly turn off auto-merging after the call, as you suggest.

Reply:
Hah I fell for another function with the same name? 🤦 Ignore me, then.